README.md

```diff
@@ -345,6 +345,29 @@ You can specify multiple labels (that option will return entities which have bot
 pods = client.get_pods(label_selector: 'name=redis-master,app=redis')
 ```
 
+Get all entities of a specific type in chunks:<br>
```

Comment:
nit: the `<br>` is unnecessary, especially as it's followed by empty line.

Reply:
Done